### PR TITLE
[ismrmrd] updated to version 1.4.2

### PR DESCRIPTION
**Pull request by dchansen**
**Describe the pull request**

- What does your PR fix? 
Updates ismrmrd to verison 14.2 which adds functionality for metacontainers and improves target based installation.

- Which triplets are supported/not supported? Have you updated the CI baseline? 
Only windows x64 and x86 have been tested

- Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)? yes

### CODE REVIEW

File: ports/ismrmrd/CONTROL [hunk 1]
```diff
@@ -1,4 +1,4 @@
 Source: ismrmrd
-Version: 1.4.1
+Version: 1.4.2
 Description: ISMRM Raw Data Format
 Build-Depends: pugixml, hdf5, boost, fftw3
```
Comment (NancyLi1013) [inline ports/ismrmrd/CONTROL:4]: Could you please add homepage here?
`Homepage: https://github.com/ismrmrd/ismrmrd`

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

**Comment by NancyLi1013** [comment]
Hi @dchansen 
Thanks for posting this issue.
I noticed that `ismrmrd `failed on x64-windows-static and x64-linux triplets.
Could you pleae try to look into them?

**Comment by dchansen** [comment]
Hi @NancyLi1013 ,
The windows static issue has been fixed.
The issue on Linux seems to be unrelated to ismrmrd. The tests indicate a problem in qt5, which is entirely in no way related.

**Review by NancyLi1013 — APPROVED** [review]

**Comment by NancyLi1013** [comment]
LGTM. Thanks for your PR @dchansen .

**Comment by ras0219-msft** [comment]
Thanks for the PR!

**Pull request merged**